x/shared/keeper/msg_server_update_param_test.go

```diff
@@ -255,7 +263,12 @@ func TestMsgUpdateParam_UpdateSupplierUnbondingPeriodSessions(t *testing.T) {
 
 // getMinSupplierUnbondingPeriodSessions returns the minimum supplier unbonding period
 // in session number that is greater than the cumulative proof window close blocks.
-func getMinSupplierUnbondingPeriodSessions(params *sharedtypes.Params, deltaBlocks uint64) uint64 {
+func getMinSupplierUnbondingPeriodSessions(
```

Comment:
I like this new helper but am a bit confused why the comment everywhere it's used is 100% the same BUT the inputs we give it are different?

1. Same function
2. Same variable assignment
3. Same comment
4. Different inputs

#PUC ? What am I misunderstanding?